includes/settings.php

```diff
@@ -454,3 +456,91 @@ function sanitize_settings( $settings ) {
 
 	return $new_settings;
 }
+
+/**
+ * Enable auto-update of plugin.
+ *
+ * @param stdClass $transient Transient cache object.
+ *
+ * @return stdClass
+ */
+function auto_update_plugins_setting( $transient ) {
+	if ( ! DT_IS_NETWORK ) {
+		$valid_license = \Distributor\Utils\get_settings()['valid_license'];
+	} else {
+		$valid_license = \Distributor\Utils\get_network_settings()['valid_license'];
```

Comment:
For both this line and the line above, we don't need the `\Distributor` namespace added, as we have a call at the beginning of this file to `use Distributor\Utils`. So both of those lines can be simplified to just `Utils\get_settings()` and `Utils\get_network_settings()`